src/components/NcGuestContent/NcGuestContent.vue

```diff
@@ -49,6 +49,12 @@ This components provides a wrapper around a guest page content.
  */
 export default {
 	name: 'NcGuestContent',
+	mounted() {
+		document.getElementById('content').classList.add('guest')
+	},
+	destroyed() {
```

Comment:
Could there be any case where more than one guest content is shown on a page?

1) render guest content a
2) render guest content b
3) remove guest content b

-> styling breaks

:imp:

Reply:
Good point. The component is meant to be used once per page as a global wrapper, similar to `NcContent`.

IMO, this shouldn't be an issue.

Reply:
I added a commit to improve the documentation of the component. Now, the usage and caveats should be clear.